src/software/ai/navigator/path_planner/hrvo/velocity_obstacle.h

```diff
@@ -0,0 +1,30 @@
+#include "software/geom/vector.h"
+
+/**
+ * A hybrid reciprocal velocity obstacle.
```

Comment:
nit: This comment should not include "hybrid reciprocal" any more as we've expanded to support regular VOs aswell.

Reply:
done